content/events/ashg2023-american-society-of-human-genetics.md

```diff
@@ -0,0 +1,32 @@
+---
+author: "AnVIL"
+conference: "ASHG 2023"
+description: "The AnVIL team will be attending the ASHG 2023 Conference to share with the broader genetics and genomics community about the AnVIL platform, scientific accomplishments it has enabled, and common genomics research challenges for which AnVIL offers solutions."
+eventType: "Conference"
+featured: true
+hashtag: "#ashg23"
+location: "Walter E. Washington Convention Center, 801 Mt Vernon Pl NW, Washington, DC 20001"
+sessions: [{sessionStart: “1 November 2023"}, {sessionStart: “2 November 2023"}, {sessionStart: “3 November 2023"}, {sessionStart: “4 November 2023"}, {sessionStart: “5 November 2023"}]
+timezone: "America/Los_Angeles"
```

Comment:
@esthermcdade would you please update the `timezone` to "America/New_York" as per ticket basics.